### PR TITLE
added new blog post about $ object

### PR DESCRIPTION
**Pull request by Graur**
#44

---

## PR-Codex overview
This PR focuses on adding a blog post about the `$` object in the EO programming language.

### Detailed summary
- Added a new blog post titled "$ Object"
- Explained the significance and functionality of the `$` object
- Provided an example to illustrate how the `$` object works
- Discussed practical applications of the `$` object
- Concluded with the importance and potential of leveraging the `$` object in EO programming

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by Graur** [comment]
@maxonfjvipon Could you please take a look?

File: _posts/2023/11/2023-11-15-this.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+date: 2023-11-15
+title: "$ Object"
+author: graur
+---
+
+In the world of EO programming, the `$` object acting as a syntax sugar that refers to the current abstract object it is used in.
+This seemingly simple yet powerful feature provides programmers with an elegant way to manipulate and access objects within their code, 
+ultimately enhancing readability and ease of use.
+In this blog post, we will dive deep into the concept of the `$` object and shed light on its practical applications through illustrative examples.
+
```
[the remark this answers is not in the document]
Comment (Graur) [reply _posts/2023/11/2023-11-15-this.md:12]: @maxonfjvipon I'm not sure I understand your proposal. Can you give me some extra hint?

[the remark this answers is not in the document]
Comment (maxonfjvipon) [reply _posts/2023/11/2023-11-15-this.md:12]: @Graur check this as [example](https://github.com/objectionary/news.eolang.org/blob/dc02ec06bfbf11c3d73c6e636263523fad33ef4b/_posts/2023/10/2023-10-04-application-to-application.md?plain=1#L16)
This delimiter is needed to hide the body of the blog post on the main page

[the remark this answers is not in the document]
Comment (Graur) [reply _posts/2023/11/2023-11-15-this.md:12]: @maxonfjvipon Now I get it, thanks!

**Comment by Graur** [comment]
@maxonfjvipon Could you please check again?

**Review by maxonfjvipon — APPROVED** [review]
@Graur goooooood one, like it, thanks!

**Comment by Graur** [comment]
@yegor256 Please have a look

**Comment by Graur** [comment]
@yegor256 Could you please take a look?

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Pull request merged**